cds_ils/migrator/utils.py

```diff
@@ -129,7 +130,32 @@ def clean_item_record(record):
     clean_item_status(record)
     clean_description_field(record)
     record["shelf"] = record["location"]
-    record["medium"] = "NOT_SPECIFIED"
+    record["medium"] = "PAPER"
     del record["location"]
     del record["id_bibrec"]
     del record["id_crcLIBRARY"]
+
+
+def clean_created_by_field(record):
+    """Clean the created by field for documents."""
+    if "_email" in record["created_by"]:
+        patron = User.query.filter_by(
+            email=record["created_by"]["_email"]
```

Comment:
Silly question: whenever I was trying this search, to filter by email field, the result was always empty. Does this really work?

Reply:
It worked but there were only 5 such records in the file I used.